network-api/networkapi/templates/pages/libraries/rcc/library_page.html

```diff
@@ -56,7 +56,7 @@ <h1>{{ page.title }}</h1>
                 </div>
                 <div class="tw-h4-heading tw-mb-0">
                     {# RESULTS COUNT #}
-                    {% comment %} {% if search_query %}
```

Comment:
The code was already here, I just had forgotten to remove a comment before :see_no_evil: